Source/Blazorise/Components/Tabs/TabPanel.razor.cs

```diff
@@ -46,6 +52,14 @@ protected override void OnInitialized()
             base.OnInitialized();
         }
 
+        /// <inheritdoc/>
+        protected override Task OnParametersSetAsync()
+        {
+            if ( Active )
+                lazyLoaded = ( Mode == TabsMode.LazyLoad );
```

Comment:
Since this is placed in `OnParametersSetAsync` isn't it's going to be executed every time a page has refreshed?

Reply:
Due to cascaded state should be pretty much every interaction with tabs. At least from my testing.
Why? This is negligible work.

Reply:
Yeah, it will work. It just felt weird at first.